cub/cub/device/dispatch/dispatch_find.cuh

```diff
@@ -0,0 +1,290 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+/**
+ * @file cub::DeviceFind provides device-wide, parallel operations for
+ *       computing search across a sequence of data items residing within
+ *       device-accessible memory.
+ */
```

Comment:
Please use `//!` style doxygen comments. Also, there needs to be a new line after `@file` and before `cub::DeviceFind`.